src/Sulu/Bundle/AdminBundle/Resources/js/components/RectangleSelection/ModifiableRectangle.js

```diff
@@ -67,26 +75,24 @@ class ModifiableRectangle extends React.Component<Props> {
 
     @action handleMouseMove = (event: MouseEvent) => {
         const {onChange} = this.props;
-
-        if (!onChange) {
-            return;
-        }
-
-        const {movementX, movementY} = event;
```

Comment:
safari's `movementX` behaves different than chrome's `movementX` does. On chrome, `movementX` stops increasing, if the cursor reached the edge of the screen. On safari, `movementX` increases as long as you scroll on the touch pad, even if the cursor already stopped at the edge of the screen.

Reply:
ofc same for `movementY`